src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -325,9 +306,9 @@ private BindingConfig GetDefaultBindingConfig()
         /// Optional hook to customize intellisense. 
         /// </summary>
         /// <returns></returns>
-        private protected virtual IIntellisense CreateIntellisense()
+        private protected virtual IIntellisense CreateIntellisense(CultureInfo culture)
```

Comment:
>CultureInfo culture) [](http://example.com/codeflow?start=67&length=20)

fyi - this will be breaking internal; all derived classes will need to be updated.

Reply:
Absolutely. I'll check PA client code